sequence_processing_pipeline/Pipeline.py

```diff
@@ -52,6 +53,11 @@ def __init__(self, configuration_file_path, run_id, sample_sheet_path,
             raise PipelineError("sample_sheet_path or mapping_file_path "
                                 "must be defined, but not both.")
 
+        if type not in ['amplicon', 'metagenomic', 'metatranscriptomic']:
```

Comment:
Could these be centralized as static strings?

`type` is a reserved word in Python. Could a variation be used, such as `type_` or `run_type`?

Reply:
Good catch, thanks!